src/state/slices/portfolioSlice/selectors.ts

```diff
@@ -946,7 +963,25 @@ export const selectPortfolioAssetIdsByAccountIdExcludeFeeAsset = createDeepEqual
   },
 )
 
-export const selectAccountIdsByAssetId = createSelector(
+export const selectAccountIdByAddress = createSelector(
```

Comment:
We don't actually use this anywhere.

Reply:
i'm going to murder this along with 1/4 of the portfolio slice in a separate PR, been waiting to avoid conflicts.

Reply:
Nooked in 783fea474874bd366fd0361109e503cd54b9ab15